### PR TITLE
Typo fixing

### PR DESCRIPTION
**Pull request by julien-lecomte**
Just some partial typo fixing with controlled codespell commands.

### CODE REVIEW

File: LICENSES/CC-BY-3.0.txt [hunk 1]
```diff
@@ -10,7 +10,7 @@ BY EXERCISING ANY RIGHTS TO THE WORK PROVIDED HERE, YOU ACCEPT AND AGREE TO BE B
 
 1. Definitions
 
-     a. "Adaptation" means a work based upon the Work, or upon the Work and other pre-existing works, such as a translation, adaptation, derivative work, arrangement of music or other alterations of a literary or artistic work, or phonogram or performance and includes cinematographic adaptations or any other form in which the Work may be recast, transformed, or adapted including in any form recognizably derived from the original, except that a work that constitutes a Collection will not be considered an Adaptation for the purpose of this License. For the avoidance of doubt, where the Work is a musical work, performance or phonogram, the synchronization of the Work in timed-relation with a moving image ("synching") will be considered an Adaptation for the purpose of this License.
```
Comment (ypid) [inline LICENSES/CC-BY-3.0.txt]: `LICENSES/*` should probably be excluded ;-) This is managed by the `reuse` command.

**Review by ypid — CHANGES_REQUESTED** [review]

File: LICENSES/CC-BY-3.0.txt (hunk 1 above)
Comment (julien-lecomte) [reply LICENSES/CC-BY-3.0.txt]: I hesitated. I'll remove this on next push.

**Review by drybjed — APPROVED** [review]
Thanks! It seems that most of these are documentation fixes, so I'll backport the PR to the stable releases and fix any merge issues.

I wonder if codespell should be added to the CI test suite... Is it feasible, or are there too many false positives without a way to mark them properly?

**Comment by julien-lecomte** [comment]
It could check but it's might be hard to auto-fix. There's a couple places with errors in licenses. I recall also one or two variables which I prefer not to touch.

**Pull request merged**